### DISCUSSION TITLE
Implementing UDP ethernet communication between PC and Teensy 4.1 using QNEthernet

### DISCUSSION THREAD

**Discussion by igor-sadalski** [ssilverman/QNEthernet#19] [Q&A] [answered]
I am trying to implement a UDP communication over Ethernet between my PC and teensy 4.1 using the QNEthernet library (sth. like UDPSendReceiveString example in NativeEthernet library also for Teensy 4.1 --> https://github.com/vjmuzik/NativeEthernet/tree/master/examples/UDPSendReceiveString) . I was wondering what would be the easiest way to implement this? Also I think this would be a nice example to have in the example folder:). kr.

**Comment by ssilverman** [comment, marked as answer]
The _QNEthernet_ UDP API is based on the Arduino UDP API. The code will be similar.
